internal/locald/rootmanager/tp_monitor.go

```diff
@@ -128,16 +129,40 @@ func (mon *tpMonitor) checkTunnelProxyAccess(ctx context.Context) bool {
 			restartSvcs = true
 		}
 	}
+	if !restartSvcs {
+		if mon.root == nil || mon.root.root == nil || mon.root.root.etcHostsSVC == nil || !mon.root.root.etcHostsSVC.Status().Healthy {
```

Comment:
1. Why are we only checking for the health of `etcHostsSVC`, instead of doing it also for `localnetSVC`?
I think both should be healthy before trying to exercise an in-cluster request.

2. Nit: we are already assuming in the code from above that `mon.root != nil`, so I don't think we need that check.

Reply:
> Why are we only checking for the health of etcHostsSVC, instead of doing it also for localnetSVC?
I think both should be healthy before trying to exercise an in-cluster request.

I've added the check.  I think they should have been checked whether or not we exercise a fresh in-cluster request -- that is likely a cause of the status reporting healthy when it is not.

anyway, thanks for the review, I've revised it.